WalletWasabi/WabiSabi/Client/CoinJoinClient.cs

```diff
@@ -659,12 +660,12 @@ private async Task ReadyToSignAsync(IEnumerable<AliceClient> aliceClients, DateT
 
 	private ImmutableList<DateTimeOffset> GetScheduledDates(int howMany, DateTimeOffset endTime)
 	{
-		return GetScheduledDates(howMany, endTime, TimeSpan.MaxValue);
+		return GetScheduledDates(howMany, DateTimeOffset.UtcNow, endTime, TimeSpan.MaxValue);
 	}
 
-	internal virtual ImmutableList<DateTimeOffset> GetScheduledDates(int howMany, DateTimeOffset endTime, TimeSpan maximumRequestDelay)
+	internal virtual ImmutableList<DateTimeOffset> GetScheduledDates(int howMany, DateTimeOffset startTime, DateTimeOffset endTime, TimeSpan maximumRequestDelay)
 	{
-		var remainingTime = endTime - DateTimeOffset.UtcNow;
+		var remainingTime = endTime - startTime;
```

Comment:
I don't understand this. GetScheduledDates is not giving back the scheduled dates. remainingTime.SamplePoisson is not taking startTime into account.

Reply:
Right. Done.